src/assets/translations/main.json

```diff
@@ -757,6 +761,17 @@
         "body": "Enter the new nickname of your wallet and your password to decrypt your wallet.",
         "button": "Next",
         "confirmDelete": "Are you sure you want to delete the nickname of this wallet?"
+      },
+      "legacy": {
+        "haveMobileWallet": "Have a ShapeShift Mobile Wallet?",
+        "loginWithAccount": "Log In with ShapeShift Account",
+        "loginInformations": "Enter your email address and password you use to login to your ShapeShift Mobile App.",
+        "emailAddress": "Email Address",
+        "password": "Password",
+        "emailRequired": "Email is required",
+        "invalidEmail": "Email is invalid",
+        "invalidCaptcha": "Captcha is invalid",
+        "invalidLogin": "Invalid login"
```

Comment:
```suggestion
        "invalidLogin": "Invalid login"
```
```suggestion
        "invalidLogin": "Invalid login",
        "twoFactorRequired": "2FA is required"
```

Reply:
Not sure why there are multiple suggestions/comments. Just wanted to suggest we'll need a 2FA error message.

Reply:
Not sure why there are multiple suggestions/comments. Just wanted to suggest we'll need a 2FA error message.